AnnoyancesFilter/sections/cookies_whitelist.txt

```diff
@@ -6,6 +6,11 @@
 @@||quantcast.mgr.consensu.org/cmp.js$domain=cnews.fr
 @@||vendorlist.consensu.org/vendorlist.json$domain=cnews.fr
 @@||static.quantcast.mgr.consensu.org/*/cmpui-banner.js$domain=cnews.fr
+! https://github.com/AdguardTeam/AdguardFilters/issues/58749
+!#if (adguard_app_ios)
+@@||vendorlist.consensu.org/vendorlist.json$domain=der-postillon.com
+@@||marfeel.mgr.consensu.org/main.tcf.cmp.js$domain=der-postillon.com
```

Comment:
Are they not required for other platforms?

Reply:
I could not reproduce this problem on other platforms